src/quicklist.c

```diff
@@ -528,14 +617,30 @@ int quicklistPushTail(quicklist *quicklist, void *value, size_t sz) {
 void quicklistAppendZiplist(quicklist *quicklist, unsigned char *zl) {
     quicklistNode *node = quicklistCreateNode();
 
-    node->zl = zl;
-    node->count = ziplistLen(node->zl);
+    node->entry = zl;
+    node->count = ziplistLen(node->entry);
     node->sz = ziplistBlobLen(zl);
 
     _quicklistInsertNodeAfter(quicklist, quicklist->tail, node);
     quicklist->count += node->count;
 }
 
+/* Create new node consisting of a pre-formed plain node.
+ * Used for loading RDBs where entire plain node has been stored
+ * to be retrieved later.
+ * zl - the data to add (pointer becomes the responsibility of quicklist) */
+void quicklistAppendPlainNode(quicklist *quicklist, unsigned char *zl, size_t sz) {
+    quicklistNode *node = quicklistCreateNode();
+
+    node->entry = zl;
+    node->count = 1;
+    node->sz = sz;
+    node->container = QUICKLIST_NODE_CONTAINER_NONE;
+
+    _quicklistInsertNodeAfter(quicklist, quicklist->tail, node);
+    quicklist->count++;
```

Comment:
```suggestion
    __quicklistInsertPlainNode(quicklist, quicklist->tail, zl, sz, 1);
```
It looks like it could just be replaced with `__quicklistInsertPlainNode`.